tools/ci_build/build.py

```diff
@@ -544,6 +544,10 @@ def parse_arguments():
     parser.add_argument('--eager_customop_header', default=None,
                         help='Header containing custom op definitions for eager mode.')
 
+    parser.add_argument(
```

Comment:
Should this be exposed in build.py given that this feature is going to be used only internally and also given the limitations the feature has (might not work on Windows, etc)?

Reply:
I don't think it should cause any problem if we document that it only works on Ubuntu.

Reply:
Added a fix in build.py to fail if the platform is not Ubuntu if --enable_external_custom_op_schemas option is specified.